client/src/components/EditDetails.jsx

```diff
@@ -1,60 +1,68 @@
-import React from 'react';
-import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
-import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css';
-import axios from 'axios';
+import React from 'react'
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap'
+import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css'
+import axios from 'axios'
+
+const Months = Array.from(Array(12)).map(i => i + 1)
+const Days = Array.from(Array(31)).map(i => i + 1)
 
 class EditDetails extends React.Component {
   constructor(props) {
-    super(props);
-    let { height, weight, age, gender, dob, zip } = props.profile;
+    super(props)
+    let { height, weight, age, gender, dob, zip } = props.profile
+
+    const [heightFt, heightIn] = height.split('/')
+
     this.state = {
-      heightFt: height.split('/')[0],
-      heightIn: height.split('/')[1],
-      weight: weight.split(' ')[0],
       age,
-      gender,
       dob,
+      gender,
+      heightFt,
+      heightIn,
       zip,
       dropdownOpen: false,
-    };
-    this.saveChanges = this.saveChanges.bind(this);
-    this.toggle = this.toggle.bind(this);
-    this.clickDropdown = this.clickDropdown.bind(this);
-    this.changeHandler = this.changeHandler.bind(this);
-  };
-
-  changeHandler(e) {
-    this.setState({ [e.target.name]: e.target.value }, () => console.log(this.state));
+      weight: weight.split(' ')[0],
+    }
   }
 
-  toggle() {
+  changeHandler = e => {
```

Comment:
Use arrow function to define a class instance method, so we don't have to `bind()` in `constructor()`

E.g.,

```ts
// BEFORE
constructor() {
  this.toggle = this.toggle.bind(this);
}

toggle() { ... }

// AFTER
toggle = () => { ... }
```